### ISSUE TITLE
Jest test are failing import { init, SearchIndex } from 'emoji-mart';

### ISSUE THREAD

**Issue by orlandodemauro** [missive/emoji-mart#743]
hello guys,
I'm using `v5.3.2` and my unit tests are failing with the below error.

Test suite failed to run

```

    ReferenceError: $d03bf5953babc97e$import$1d20cf026fb5935d$2e2bcd8739ae039 is not defined

    import { init, SearchIndex } from 'emoji-mart';
         | ^
       9 |
```

not sure if it is a duplicate of #734 because i can reproduce the issue also downgrading to `v5.2.2`

thanks

**Comment by keven1024** [comment]
The same problem  #742, I downgraded the version to `v5.2.0` everything is fine

**Comment by EtienneLem** [comment]
Should be fixed in **v5.3.3**
